musings/workload-identity.md

```diff
@@ -33,7 +33,10 @@ The workload identity problem space can be defined by the following set of key s
    * For long-lived workloads, the lifetime of a workload is likely to exceed that of the credentials issued to it, requiring ability to renew and/or revoke credentials in response to governance requirements and/or emerging threats without compromising overall system reliability
 9. Governance of workload identities and credentials must include some of the following capabilities:
    * Maintaining control over and a history of all policies and decisions involved in issuing and evaluating identities and credentials
-    Ability to identify, isolate and remedy vulnerable workloads in production quickly, such as when new threats are discovered or new attacks detected
+   * Ability to identify, isolate and remedy vulnerable workloads in production quickly, such as when new threats are discovered or new attacks detected
+10. A workload's identity should coexist in a predictable manner regular datacenter maintenance operations such as workload migration
+   * Operations that would change the workload identity (e.g., by modifying one of its claims) should require redeployment where the old workload is torn down and the new one stood up in its place with a new workload identity
```

Comment:
I'm not sure about this. If an attribute of the workload's TCB changes but is not relevant from an identification point of view, I don't think there is a good reason for "redeployment," as described.

Reply:
Perhaps we should water down the requirement, as part of what we have written is true as well, but not necessarily all the time!

Reply:
Any claim can be thought of as relevant from the identification point of view. You never know which claims the relying party will use to allow or deny access, so you can't just continue a workload if its claims change.